debug/test_bulk_generate.py

```diff
@@ -0,0 +1,15 @@
+import os

```

Comment:
Utility function I want to keep around, useful for testing rooms and getting an idea of how often they fail to gen